packages/language-service/src/baseLanguageService.ts

```diff
@@ -62,11 +62,7 @@ function createLanguageServicePluginContext(
 		const created = tsFaster.createLanguageService(
 			ts,
 			languageContext.typescript.languageServiceHost,
-			(proxiedHost) => {
-				// commented out for fixed write virtual files not working
-				// languageContext.typescript.languageServiceHost = proxiedHost;
-				return ts.createLanguageService(proxiedHost, documentRegistry);
-			},
```

Comment:
Tested that the proxied host is not needed so removed the comments.

Reply:
Thanks for confirming that.